dozer/cogs/moderation.py

```diff
@@ -75,11 +75,12 @@ class Moderation(Cog):
         except:
             minutes = 0
         time = (hours * 3600) + (minutes * 60)
-        if time == 0:
+        if time == 0 and modlog:
             await self.mod_log(member=ctx.author, action=punishment.past_participle, target=target, reason=reason, orig_channel=ctx.channel)
         else:
             reasoning = re.sub(pattern=regex_string, string=reason, repl="").lstrip("  ")
-            await self.mod_log(member=ctx.author, action=punishment.past_participle, target=target, reason=reasoning, orig_channel=ctx.channel)
+            if modlog:
```

Comment:
So you've got the `modlog` check in the initial if statement, the you've got another one here... there's got to be a cleaner way of doing this.

Reply:
Neither `reason` nor `reasoning` are used after this point, so you can reassign `reason` here and then do the log after this if suite.